cura/CuraApplication.py

```diff
@@ -1600,8 +1600,11 @@ def log(self, msg):
 
     openProjectFile = pyqtSignal(QUrl, arguments = ["project_file"])  # Emitted when a project file is about to open.
 
-    @pyqtSlot(QUrl, bool)
-    def readLocalFile(self, file, skip_project_file_check = False):
+    @pyqtSlot(QUrl, str)
+    ## Open a local file
+    # \param force_project_mode \type{str} Either "open_as_model" or "open_as_project". This parameter is only
+    # considered if the file is a project file.
+    def readLocalFile(self, file, force_project_mode = None):
```

Comment:
Please provide typing in the parameter rather than in the documentation. If you're looking at other places where it was in the documentation, that comes from legacy from before we were using Python 3.5. If it's in the parameter, MyPy can read it.

When providing the typing, MyPy will also start complaining that it should be `Optional[str]` rather than just `str`.